app/views/tufts/_workflow_status_badge.html.erb

```diff
@@ -0,0 +1,7 @@
+<% status = Tufts::WorkflowStatus.status(locals[:id]) %>
```

Comment:
I'm thinking pass through the presenter, instead of just the id. Then the call is `presenter.status` or similar.
When we implement indexing, the presenter can be refactored to simply read the solr document.